src/plugins/auth.ts

```diff
@@ -63,7 +64,8 @@ async function refreshToken(): Promise<User | undefined> {
     const user = await userManager.signinSilent();
 
     const newUser: User = {
-      access_token: user?.access_token || '',
+      access_token:
```

Comment:
Should we call the field just `token` now?
In the if conditition, lets use the `id_token` only if `env.idpTokenType=='id_token'` to have forward compatability.